### PR TITLE
2021+ Rav4 Prime

### PR DESCRIPTION
**Pull request by pd0wm**
Example Route: `20ba9ade056a8c7b|2021-02-08--21-57-35`

- Missing are `BRAKE_MODULE`, `GEAR_PACKET`, `PCM_CRUISE`. `KINEMATICS` is also missing, but not used by openpilot.
- Messages with SecOC MAC: `STEERING_LKA`, `LTA_RELATED` (related to LTA based on FW analysis, no LTA drive seen yet. `STEERING_LTA` does not have SeOC), `PRE_COLLISION` (The `STATE` field is now occupied by the MAC. How does the new layout work?). `PRE_COLLISION_2`.
- Remove `GEAR_PACKET_HYBRID.COUNTER`, it's not a counter and fails some checks in openpilot. Also see this corolla hybrid route `075b133b6181e058|2024-01-26--10-39-08`.

### CODE REVIEW

File: generator/toyota/toyota_nodsu_secoc_pt.dbc [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+ SG_ BRAKE_PRESSED : 3|1@0+ (1,0) [0|1] "" XXX
+ SG_ BRAKE_PRESSURE_1 : 31|8@0+ (1,0) [0|15] "" XXX
+ SG_ BRAKE_PRESSURE_2 : 61|6@0+ (1,0) [0|63] "" XXX
+
+BO_ 305 LTA_RELATED_2: 8 XXX
+ SG_ COUNTER : 13|6@0+ (1,0) [0|63] "" XXX
+ SG_ STEERING_ANGLE_RELATED : 23|16@0- (1,0) [0|65535] "" XXX
+ SG_ AUTHENTICATOR : 35|28@0+ (1,0) [0|268435455] "" XXX
+ SG_ RESET_FLAG : 37|2@0+ (1,0) [0|3] "" XXX
+ SG_ MSG_CNT_LOWER : 39|2@0+ (1,0) [0|3] "" XXX
```
Comment (sshane) [inline generator/toyota/toyota_nodsu_secoc_pt.dbc:22]:
```suggestion
BO_ 305 LTA_RELATED_2: 8 XXX
 SG_ STEER_REQUEST_1 : 0|1@0+ (1,0) [0|1] "" XXX
 SG_ STEER_REQUEST_2 : 3|1@0+ (1,0) [0|1] "" XXX
 SG_ COUNTER : 13|6@0+ (1,0) [0|63] "" XXX
 SG_ STEERING_ANGLE_CMD : 23|16@0- (0.0573,0) [0|65535] "" XXX
 SG_ AUTHENTICATOR : 35|28@0+ (1,0) [0|268435455] "" XXX
 SG_ RESET_FLAG : 37|2@0+ (1,0) [0|3] "" XXX
 SG_ MSG_CNT_LOWER : 39|2@0+ (1,0) [0|3] "" XXX
```

**Comment by jyoung8607** [comment]
Superseded by #1287.

**Comment by dgofman** [comment]
I’m curious to know if the Comma 3X is compatible with the 2021 RAV4 Prime.
Additionally, could you please direct me to any resources or information regarding the installation process for this device in my car?

**Comment by jyoung8607** [comment]
The answer is a provisional yes, it's a little complicated.

More info at https://discord.comma.ai in #toyota-security.

**Pull request closed without merging**